libs/service/discord/src/getDiscordBot.ts

```diff
@@ -15,7 +19,42 @@ export const getDiscordBot = (token: string): Promise<DiscordClient> =>
 		try {
 			bot.login(token);
 
-			bot.on("ready", () => resolve(bot));
+			bot.once("ready", () => resolve(bot));
+		} catch (error) {
+			reject(error);
+		}
+	});
+
+export type DiscordWebhooks = [InteractionWebhook, InteractionWebhook];
+
+export const getDiscordWebhooks = (
+	token: string,
+	channelIds: [string, string],
+	[proposalWebhookId, referendumWebhookId]: [string, string]
```

Comment:
this logic is fine, but you can tweak so it can receive an array of `webhookIds` which will return an array of `InteractionWebhook`